projects/doc-lib/src/vanilla/README.md

```diff
@@ -0,0 +1,3 @@
+# Vanilla JS Components
```

Comment:
vanilla as a folder name is a little too abstract for me. When I look into the code base as a new bee, It’s hard for me to understand what’s in there. Maybe… plain-js/

Reply:
Renamed to plain-js-components